.circleci/config.yml

```diff
@@ -72,8 +71,8 @@ jobs:
           echo "//npm.pkg.github.com/:_authToken=${GITHUB_PACKAGES_WRITE_TOKEN}" > ~/.npmrc
           echo "@contentful:registry=https://npm.pkg.github.com" >> ~/.npmrc
       - run: yarn build # This should not be needed, but without it it will not contain any files
-      - run: yarn lerna version --no-private --conventional-commits --conventional-prerelease
-      - run: yarn lerna publish from-git --yes
+      - run: yarn version:canary
+      - run: yarn publish:canary
```

Comment:
should this be removed?

Reply:
no this would only be triggered if we are merging to the canary branch which we might do for future iterations of live preview sdk & csm but it won't be used by this PR instead we will trigger the job release on line 47